styles/styles.js

```diff
@@ -21,6 +21,14 @@ module.exports = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
   },
+  header: {
```

Comment:
C'è un motivo per cui questi stili non sono nei rispettivi moduli? Sono stili globali? Just out of curiosity